### PR TITLE
[ISSUES-3063][Improve] Improve ImplicitsUtilsTest test code coverage

### PR DESCRIPTION
**Pull request by zhengkezhou1**
## What changes were proposed in this pull request
Improve streampark-common module unit test 

## Verifying this change

This change added tests and can be verified as follows:

## Does this pull request potentially affect one of the following parts
 - Dependencies (does it add or upgrade a dependency): (no)

### CODE REVIEW

File: streampark-common/src/test/scala/org/apache/streampark/common/util/ImplicitsUtilsTest.scala [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import org.apache.streampark.common.util.ImplicitsUtils.{AutoCloseImplicits, StringImplicits}
+
+import org.scalatest.funsuite.AnyFunSuite
+
+class ImplicitsUtilsTest extends AnyFunSuite {
+  test(
+    "AutoCloseImplicits.autoClose should close the resource after execution and handle exceptions") {
+    class MockResource extends AutoCloseable {
+      var closed = false
+      def close(): Unit = closed = true
```
Comment (wolfboys) [inline streampark-common/src/test/scala/org/apache/streampark/common/util/ImplicitsUtilsTest.scala:29]: good job😂

**Review by wolfboys — APPROVED** [review]
LGTM

**Comment by caicancai** [comment]
@zzzk1 @wolfboys Hello, I think some negative tests should be added here, and I will submit a PR to improve it.

File: streampark-common/src/test/scala/org/apache/streampark/common/util/ImplicitsUtilsTest.scala (hunk 1 above)
Comment (caicancai) [reply streampark-common/src/test/scala/org/apache/streampark/common/util/ImplicitsUtilsTest.scala:29]: +1

**Pull request merged**